intertidal/elevation.py

```diff
@@ -147,169 +169,463 @@ def ds_to_flat(ds, ndwi_thresh=0.1, index='ndwi', min_freq=0.01, max_freq=0.99,
     return ds_flat, freq, good_mask
 
 
-def create_dask_gateway_cluster(profile="r5_L", workers=2):
-    """
-    Create a cluster in our internal dask cluster.
-    Parameters
-    ----------
-    profile : str
-        Possible values are:
-            - r5_L (2 cores, 15GB memory)
-            - r5_XL (4 cores, 31GB memory)
-            - r5_2XL (8 cores, 63GB memory)
-            - r5_4XL (16 cores, 127GB memory)
-    workers : int
-        Number of workers in the cluster.
+def rolling_tide_window(
+    i,
+    ds,
+    window_spacing,
+    window_radius,
+    tide_min,
+    statistic="median",
+):
     """
+    This function takes a rolling window of tide observations from
```

Comment:
To do in the future: update all docstrings using the numpy style guide for continuity